### PR TITLE
Broadcast source unit test inval delete

### PR DESCRIPTION
**Pull request by Thalley**

### CODE REVIEW

**Review by larsgk — DISMISSED** [review]

**Review by cvinayak — DISMISSED** [review]

**Comment by fabiobaltieri** [comment]
@Thalley also merge conflict, needs a rebase

**Comment by Thalley** [comment]
Rebased to solve merge conflicts

File: tests/bluetooth/audio/bap_broadcast_source/src/main.c [hunk 1]
```diff
@@ -268,3 +268,35 @@ ZTEST_F(bap_broadcast_source_test_suite, test_broadcast_source_start_inval_doubl
 	err = bt_bap_broadcast_source_start(fixture->source, &ext_adv);
 	zassert_not_equal(0, err, "Did not fail with starting already started source");
 }
+
```
Comment (cvinayak) [inline tests/bluetooth/audio/bap_broadcast_source/src/main.c:271]:
```suggestion
```

**Review by cvinayak — CHANGES_REQUESTED** [review]

**Review by cvinayak — DISMISSED** [review]

**Review by larsgk — DISMISSED** [review]

**Comment by Thalley** [comment]
Rebased to solve merge conflicts

**Comment by carlescufi** [comment]
@cvinayak @larsgk please re-review

**Comment by Thalley** [comment]
Rebased to solve merge conflicts

**Review by cvinayak — APPROVED** [review]

**Review by larsgk — APPROVED** [review]

**Pull request merged**